docs/guides/setup-channel.mdx

```diff
@@ -205,50 +243,50 @@ Alternatively, you can also access quantum processors with REST APIs, enabling y
     1. Find your API key. From the [API keys page](https://cloud.ibm.com/iam/apikeys), view or create your API key, then copy it to a secure location so you can use it for authentication.
     2. Find your Cloud Resource Name (CRN). Open the [Instances page](https://cloud.ibm.com/quantum/instances) and click your instance. In the page that opens, click the icon to copy your CRN. Save it in a secure location so you can use it for authentication.
 
-<span id="cloud-save"></span>
-1. Authenticate to the service by calling `QiskitRuntimeService` with your saved credentials or with your IBM Cloud API key and CRN:
+1. <span id="cloud-save"></span>**If you are working in a trusted Python environment (such as on a personal laptop or workstation),** use the `save_account()` method to save your credentials locally. ([Skip to the next step](#cloud-untrusted) if you are not using a trusted environment, such as a shared or public computer, to authenticate to IBM Cloud.) To use `save_account()`, activate a Python virtual environment (see the [installation instructions](/guides/install-qiskit#local) if you do not already have a virtual environment set up). Then, run Python in your virtual environment to open a read-eval-print loop (REPL), and enter the following:
 
-   ```python
-   from qiskit_ibm_runtime import QiskitRuntimeService
+    ```python
+    from qiskit_ibm_runtime import QiskitRuntimeService
 
-   service = QiskitRuntimeService(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>")
-   ```
+    service = QiskitRuntimeService(
+    channel="ibm_cloud",
+    token="<IBM Cloud API key>", # Your token is confidential.
```

Comment:
Same as my comment for IQP setup, I'd separate out 

```python
token = <IBM Cloud API key>
```

from the rest, so it's easy to edit in REPL.

Reply:
done in 6f5da2e749e1e5648bdf8fa9cde467cd7c756be5